gender_novels/analysis/dependency_parsing.py

```diff
@@ -27,63 +28,40 @@ def get_parser(path_to_jar, path_to_models_jar):
     return parser
 
 
-def count_gender_subj_obj(triples):
+def count_gender_subj_obj(tree):
     """
-    This function takes in a list of dependency triples for a sentence and counts female and male
-    subject and object occurrences
+    This function takes in a tree of dependency triples for a list of sentence and counts female
+    and male subject and object occurrences
 
     We have chosen not to include indirect object positions because whether or not they represent
     passivity (at least, compared to being a direct object) is debatable
 
-    :param triples: A list of dependency triples
+    :param tree: A tree containing a list of dependency triples for each sentence
     :return: the counts of male and female subject and object occurrences as a tuple of 4
```

Comment:
Can you add the order of the numbers -- since they're all "1" it's hard to tell from the test or docs what the order is.